### ISSUE TITLE
Please Add Signal

### ISSUE THREAD

**Issue by motivo1** [meetfranz/plugins#465]
### Service

Signal Private Messenger
Millions of people use Signal every day for free and instantaneous communication anywhere in the world. Send and receive high-fidelity messages, participate in HD voice/video calls, and explore a growing set of new features that help you stay connected. Signal’s advanced privacy-preserving technology is always enabled, so you can focus on sharing the moments that matter with the people who matter to you.

**Comment by hinestechworkacct** [comment]
Seconded! I'm not smart enough to write it or I'd do it myself!

**Comment by simonemonti1** [comment]
I agree! add Signal!

**Comment by deasterdaywfp** [comment]
yes agreed

**Comment by thegoof13** [comment]
Agree, this would be a great advantage and I know a few of my tech friends would get franz if it was available. Currently have to use a seperate App on my mac.

**Comment by wopophunk75** [comment]
> Agree, this would be a great advantage and I know a few of my tech friends would get franz if it was available. Currently have to use a seperate App on my mac.

exactly the same here!!!

**Comment by mingxun** [comment]
agree

**Comment by omerpines** [comment]
+1

**Comment by PiFlUn** [comment]
+1

**Comment by realdannys** [comment]
Ad another +1

**Comment by Ralphi67** [comment]
what about SIGNAL?? please add!!!

**Comment by WKyc1990** [comment]
Please add Signal. Thank you.

**Comment by rgcouto** [comment]
+1

**Comment by DieEgg** [comment]
+1 !

**Comment by people-can-fly** [comment]
Please add Signal

**Comment by noly** [comment]
Please add Signal 🙏

**Comment by DeN-AlB** [comment]
+1 for Signal

**Comment by PrimozR** [comment]
If you need somebody to test Signal, I'm up for it.

**Comment by chegwa** [comment]
+1
